Rawl_B502/bunad_laigen.ttl

```diff
@@ -192,270 +239,383 @@
-    irishRel:nomName "Gomér" .
+    irishRel:nomName "Gomér" ;
+    owl:sameAs <http://example.com/Rawl_B502/fursunnud_naile.ttl#Gomér>;
+    rel:ancestorOf <#NuaduNecht>.
 
 <#Iaféth>
     a foaf:Person ;
     rdfs:seeAlso <https://www.wikidata.org/entity/Q200637> ;
-    irishRel:nomName "Iaféth" .
+    irishRel:nomName "Iaféth" ;
+    owl:sameAs <http://example.com/Rawl_B502/fursunnud_naile.ttl#Iaféth>;
+    rel:ancestorOf <#NuaduNecht>.
 
 <#Noe>
     a foaf:Person ;
     rdfs:seeAlso <https://www.wikidata.org/entity/Q81422> ;
-    irishRel:nomName "Noe" .
+    irishRel:nomName "Noe" ;
+    owl:sameAs <http://example.com/Rawl_B502/fursunnud_naile.ttl#Nóe>;
+    rel:ancestorOf <#NuaduNecht>.
 
 <#Lamiach>
     a foaf:Person ;
     rdfs:seeAlso <https://www.wikidata.org/entity/Q10921342> ;
-    irishRel:nomName "Lamiach" .
+    irishRel:nomName "Lamiach" ;
+    owl:sameAs <http://example.com/Rawl_B502/fursunnud_naile.ttl#Lamiach>;
+    rel:ancestorOf <#NuaduNecht>.
 
 <#Mathusalám>
     a foaf:Person ;
     rdfs:seeAlso <https://www.wikidata.org/entity/Q156290> ;
-    irishRel:nomName "Mathusalám" .
+    irishRel:nomName "Mathusalám" ;
+    owl:sameAs <http://example.com/Rawl_B502/fursunnud_naile.ttl#Él>;
+    rel:ancestorOf <#NuaduNecht>.
 
 <#Enóc>
     a foaf:Person ;
     rdfs:seeAlso <https://www.wikidata.org/entity/Q213027> ;
-    irishRel:nomName "Enóc" .
+    irishRel:nomName "Enóc" ;
+    owl:sameAs <http://example.com/Rawl_B502/fursunnud_naile.ttl#Enóc>;
+    rel:ancestorOf <#NuaduNecht>.
 
 <#Iaréth>
     a foaf:Person ;
     rdfs:seeAlso <https://www.wikidata.org/entity/Q927410> ;
-    irishRel:nomName "Iaréth" .
+    irishRel:nomName "Iaréth" ;
+    owl:sameAs <http://example.com/Rawl_B502/fursunnud_naile.ttl#Iaréth>;
+    rel:ancestorOf <#NuaduNecht>.
 
 <#Malalél>
     a foaf:Person ;
     rdfs:seeAlso <https://www.wikidata.org/entity/Q10297002> ;
-    irishRel:nomName "Malalél" .
+    irishRel:nomName "Malalél" ;
+    owl:sameAs <http://example.com/Rawl_B502/fursunnud_naile.ttl#Malalél>;
+    rel:ancestorOf <#NuaduNecht>.
 
 <#Cáinán>
     a foaf:Person ;
     rdfs:seeAlso <https://www.wikidata.org/entity/Q844433> ;
-    irishRel:nomName "Cáinán" .
+    irishRel:nomName "Cáinán" ;
+    owl:sameAs <http://example.com/Rawl_B502/fursunnud_naile.ttl#Cáinán>;
+    rel:ancestorOf <#NuaduNecht>.
 
 <#Enos>
     a foaf:Person ;
     rdfs:seeAlso <https://www.wikidata.org/entity/Q682531> ;
-    irishRel:nomName "Enos" .
+    irishRel:nomName "Enos" ;
+    owl:sameAs <http://example.com/Rawl_B502/fursunnud_naile.ttl#Enos>;
+    rel:ancestorOf <#NuaduNecht>.
 
 <#Séth-44de4e08>
     a foaf:Person ;
     rdfs:seeAlso <https://www.wikidata.org/wiki/Q107626> ;
-    irishRel:nomName "Séth" .
+    irishRel:nomName "Séth" ;
+    owl:sameAs <http://example.com/Rawl_B502/fursunnud_naile.ttl#Séth>;
+    rel:ancestorOf <#NuaduNecht>.
     #same as <#Séth> ? CY
+    # I don't think so. They're in very different positions in the pedigree. EPT
```

Comment:
Hmm. Shall we say that one of us comments, the other makes a response, then the first reviews, then either makes/omits to make the relevant change or creates an issue on Git, depending on whether the issue has been resolved?